File: HW1/Circle.java
public class Circle extends Shape {

	private int r; // r- radius

	public Circle (int nx, int ny, int nz, int nr) {
		super ("circle", nx, ny, nz);
		r = nr;
	}

	public void display() {
		super.display();
		System.out.println ("radius : " + r  + " ]"); 
	}

}

File: HW2/Circle.py
from Shape import *

class Circle (Shape):

  	def __init__(self, nx, ny, nz, nr):
  		super(Circle, self).__init__("circle", nx, ny, nz)
  		self._r = nr

  	def getArea(self):
  		return 3.1416*self._r * self._r;

	def display(self):
		super(Circle, self).display()
		print("[ radius : " + str(self._r) + " ]")
